test/OpenTelemetry.Instrumentation.SqlClient.Tests/SqlEventSourceTests.netfx.cs

```diff
@@ -153,7 +153,7 @@ public void EventSourceFakeUnknownEventWithNullPayloadTest()
 
             fakeSqlEventSource.WriteUnknownEventWithNullPayload();
 
-            Assert.Equal(0, activityProcessor.Invocations.Count);
+            Assert.InRange(activityProcessor.Invocations.Count, 0, 1);
```

Comment:
try expliitly calling shutdownSignal.Dispose. It'll be confusing in the future as to why the behavior of this test is undeterministic , so lets try to fix the actual issue.